store/mockstore/mocktikv/rpc.go

```diff
@@ -736,12 +739,18 @@ func (c *RPCClient) SendRequest(ctx context.Context, addr string, req *tikvrpc.R
 		}
 	})
 
+	reqCtx := &req.Context
+	resp := &tikvrpc.Response{}
+	// When the store type is TiDBMem, the request should handle over to TiDB rpc server to handle.
+	if req.Type == tikvrpc.CmdCop && req.StoreTp == kv.TiDBMem && TiDBRPCServerCoprocessorHandler != nil {
```

Comment:
It's better to define the hijack function as a field of `RPCClient` instead of a global variable to make the codebase flexible (e.g some test cases overwrite the same global variable).